docs/vocabulary.md

```diff
@@ -178,16 +178,16 @@ k8s-style resources for declaring CI/CD-style pipelines.
 Some of the core Tekton terms are listed below. [[8]]
 
 - **Step**: a specific function to perform.
-- **Task**: is a collection of sequential steps you would want to run as part of your
-continuous integration flow. A task will run inside a pod on your cluster.
+- **Task**: is a collection of sequential steps you would want to run as part
+of your continuous integration flow. A task will run inside a pod on your cluster.
 - **ClusterTask**: Similar to Task, but with a cluster scope.
-- **Pipeline**: stateless, reusable, paramterized collection of tasks. Tasks linked
-- **PipelineRun**: running instantiations of Pipelines
-by input and output PipelineResources
-- **Pipeline Resource**: input to or output from tasks
-- **Trigger**: Triggers is a Kubernetes Custom Resource Defintion (CRD) controller
-that allows you to extract information from events payloads (a "trigger") to create
-Kubernetes resources.
+- **Pipeline**: stateless, reusable, parameterized collection of tasks. Tasks are
```

Comment:
Rather than 'collection of tasks', consider 'ordered series of tasks'

Reply:
Updated, thanks @MarckK !